sdk/tables/azure-data-tables/src/table_clients.cpp

```diff
@@ -128,24 +90,6 @@ TableClient TableServiceClient::GetTableClient(
   throw std::runtime_error("TableServiceClient is not properly initialized.");
 }
 
-TableServiceClient TableServiceClient::CreateFromConnectionString(
-    const std::string& connectionString,
-    const TableClientOptions& options)
-{
-  auto parsedConnectionString = ParseConnectionString(connectionString);
```

Comment:
We no longer need these connection string helpers and can remove them:
https://github.com/Azure/azure-sdk-for-cpp/blob/384552adff3d6cdea46cdf8a345120b183bdcc1e/sdk/tables/azure-data-tables/inc/azure/data/tables/credentials/named_key_credential.hpp#L72-L83

Reply:
Tracking in https://github.com/Azure/azure-sdk-for-cpp/issues/6267